src/services/GLSL/vertex_shader_catalog.glsl

```diff
@@ -69,6 +70,18 @@ float getSquareSideByArea(float area, float minorSize) {
     }
 }
 
+float getSize(float pointSize) {
```

Comment:
I suggest we include info of "image pixel mode" in the function naming, e.g. `getImagePixelSize`.